hack/kiali/install-ossm-release.sh

```
@@ -0,0 +1,267 @@
+fi
+if [[ "${OC}" = *"oc" ]]; then
+  # if we are using 'oc' (as opposed to 'kubectl') we therefore assume we are installing in an OpenShift cluster
+  IS_OPENSHIFT="true"
+  OLM_OPERATORS_NAMESPACE="openshift-operators"
+  if ! ${OC} whoami >& /dev/null; then
+    errormsg "You are not logged into the OpenShift cluster. Use '${OC} login' to log into a cluster and then retry."
+    exit 1
+  fi
+else
+  IS_OPENSHIFT="false"
+  OLM_OPERATORS_NAMESPACE="operators"
```

Shellcheck (0.11.0)

[warning] OLM_OPERATORS_NAMESPACE appears unused. Verify use (or export if used externally).

(SC2034)